docs/components/modeler/desktop-modeler/telemetry/telemetry.md

```diff
@@ -54,8 +54,8 @@ These events include the following properties:
 
 - `diagramType`: BPMN, DMN, or Form
 - Engine profile:
-  - `executionPlatform`: <target platform\>
-  - `executionPlatformVersion`: <target platform version\>
+  - `executionPlatform`: &lt;target platform\>
```

Comment:
Interesting... something to take to the tech writers about how we want to format info like this.

Reply:
Yeah...we are not universal in how we present "a value that is represented by something we can describe". Quotes, curly braces, angle brackets....those seem to be the three ways I can remember seeing.